src/main/scala/scorex/core/NodeViewModifier.scala

```diff
@@ -37,7 +37,7 @@ object NodeViewModifier {
 }
 
 
-trait PersistentNodeViewModifier extends NodeViewModifier {
+trait PersistentNodeViewModifier extends NodeViewModifier with JsonSerializable {
```

Comment:
Why are you adding `JsonSerializable` here, if the goal of this PR is to remove it?

Reply:
I added it to use `json` function to use when converting generic `Transaction` types to json. 
I thought its better to use `JsonSerializable` rather than adding `json` function to `Transaction` and `PersistentNodeViewModifier` traits.

https://github.com/ScorexFoundation/Scorex/blob/05cf058faf650cb67dcd6adb6441dbde5922e67a/src/main/scala/scorex/core/api/http/NodeViewApiRoute.scala#L64

If is it ok to add `json` function to `Transaction` and `PersistentNodeViewModifier` then we can remove `JsonSerializable` completely.

Reply:
What have we achieved then? Just def this = as.Json for modifiers  which is not looking good